File: pc-editor/src/main/java/com/auto_persona/pc/model/AppSettings.java
package com.auto_persona.pc.model;

import java.util.Map;

public class AppSettings {
    public int bgmVolume = 70;
    public int voiceVolume = 80;
    public boolean isMuted;
    public Map<String, String> longTermMemory = Map.of();
    public boolean ttsEnabled = true;
    public boolean presetRepliesEnabled = true;
}

File: app/src/main/java/com/example/auto_persona/data/model/AppSettings.kt
package com.example.auto_persona.data.model

import androidx.compose.runtime.Immutable
import kotlinx.serialization.Serializable

@Immutable
@Serializable
data class AppSettings(
    val bgmVolume: Int = 70,
    val voiceVolume: Int = 80,
    val isMuted: Boolean = false,
    val longTermMemory: Map<String, String> = emptyMap(),
    val ttsEnabled: Boolean = true,
    val presetRepliesEnabled: Boolean = true
)
